data/wanderers start.txt

```diff
@@ -376,18 +376,84 @@ mission "Tell Wanderers about the Pug"
 				decline
 
 
+mission "Wanderers: Defend Vara Ke'sok Hint"
+	name "Wanderer supplies to <planet>"
+	description "Transport <cargo> to <destination>, where repairs are needed from Unfettered attacks."
+	passengers 1
+	cargo "building supplies" 10
+	source 
+		near "Sich'ka'ara" 1 100
+	destination "Vara Ke'sok"
+	to offer
+		random < 60
+		has "Wanderers Conversation: offered"
+		not "Wanderers: Defend Vara Ke'sok: done"
+	on offer
+		conversation
+			`Walking around the spaceport, you are approached by two Wanderers. One of them begins speaking in the Hai language, which your translator picks up. "Greetings, [stranger, curiosity]. We heard you perform [charitable, heroic] acts. Is this correct?"`
+			choice
+				`	"Yes it is. What do you need?"`
+					goto proposal
+				`	"Depends on how much you are willing to pay."`
+					goto jerk
+				`	"Where did you hear that from?"`
+			branch hai
+				"reputation: Hai" > 100
+			branch war
+				has "main plot completed"
+			`	"Ambassador Sayari mentioned humans do many [jobs, hobbies] in their space. Or are we mistaken?"`
+			choice
+				`	"Humans aren't too altruistic, but I'd be happy to help."`
+					goto proposal
+				`	"We only do those jobs to get paid."`
```

Comment:
Neither of these seem like positive options. In both you're basically throwing shade at all of humanity. I'd drop the "Human's aren't too altruistic" part from the first one. You're basically the ambassador of humanity to the Wanderers. Don't want to make your people look bad.

Reply:
Made the first reply more positive for humanity.